packages/framework/lib/handlers/text.ts

```diff
@@ -35,13 +35,15 @@ export default function textHandlerFactory<U extends User<any>>(
 ) {
     function textHandler(
         this: Bot<U>,
-        message: { text: string },
+        message: { text: string; tracking_data?: { [key: string]: string } | string },
```

Comment:
`{ [key: string]: string } | string`

this type is wrong for tracking_data because what if tracking data is this: 

`{ "test": 1 }`

or what if tracking data is this:

`{ "test": { "test2": "ok" } }`

a better type would be a recursive one such as this: 

```typescript
export type TrackingData = Record<string, number | string | TrackingData> | string
```

Reply:
Updated the comment to add `boolean` on the recursive type

Reply:
also array type should be added...

Reply:
```typescript
export type TrackingData = Record<string, number | string | boolean | Array<TrackingData> | TrackingData>
```

Reply:
Can I just use any instead?

Reply:
no

Reply:
any would also be other types...

Reply:
eg: 

Map, Set, Function, Symbol, and whatever else you might imagine...